.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: python
-cache: pip
+cache:
+  directories:
+    - /home/travis/.rvm/gems
```

Comment:
**praise:** 👏 Thank you for thinking about optimizations and lookin into it @aireilly